mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -151,13 +151,18 @@ static bool isSupportedCombiningKind(CombiningKind combiningKind,
   return false;
 }
 
-AffineMap mlir::vector::getTransferMinorIdentityMap(ShapedType shapedType,
-                                                    VectorType vectorType) {
-  int64_t elementVectorRank = 0;
+static unsigned getRealVectorRank(ShapedType shapedType,
```

Comment:
Please document this method. In particular, what's "real vector rank" in this context? Thanks!